docs/resources/endpoint_tag_ip.md

```diff
@@ -112,3 +125,7 @@ import {
   to = aci_endpoint_tag_ip.example_tenant
 }
 ```
+
+## Child Resources
+  
+  - [aci_tag](https://registry.terraform.io/providers/CiscoDevNet/aci/latest/docs/resources/tag)
```

Comment:
Why do we have this for aci_tag and not for annotation?

Reply:
Because I forgot to exclude it. Don't think we need the reference here because the child is already included into the resource configuration. This was intended for non relational children. Shall I exclude tag also or you want to include annotation?

Reply:
exclude is fine.